### PR TITLE
add log when starting workspace with pvc

### PR DESCRIPTION
**Pull request by sagor999**
## Related Issue(s)

Related https://github.com/gitpod-io/gitpod/issues/11823

## Release Notes

```release-note
none
```

## Werft options:

- [ ] /werft with-preview

### CODE REVIEW

File: components/server/src/workspace/workspace-starter.ts [hunk 1]
```diff
@@ -1366,6 +1366,16 @@ export class WorkspaceStarter {
...
+            log.info(
+                {
+                    lastInstanceId: lastValidWorkspaceInstanceId,
+                    volSnapshotId: volumeSnapshotId,
+                    snapshotId: volumeSnapshots.id,
+                    volumeHandle: volumeSnapshots.volumeHandle,
+                    workspaceId: workspace.id,
+                },
+                "starting workspace with volume snapshot info",
+            );
```
Comment (andrew-farries) [inline components/server/src/workspace/workspace-starter.ts:1378]: would a metric be more useful than a log message or is there already something in place?

**Review by andrew-farries — APPROVED** [review]
/hold for question about log vs metric

File: components/server/src/workspace/workspace-starter.ts (hunk 1 above)
Comment (sagor999) [reply components/server/src/workspace/workspace-starter.ts:1378]: No, we kind of need log instead for debugging reasons.

**Pull request merged**